app/controllers/order_imports_controller.rb

```diff
@@ -28,6 +28,14 @@ def create
     redirect_to new_facility_order_import_path
   end
 
+  def error_report
+    if order_import.error_file_present?
+      redirect_to order_import.error_file_download_url
+    else
+      render_404
```

Comment:
Do we want to call this directly, or `raise ActiveRecord::RecordNotFound`? What do we do elsewhere?